ui/src/config/inversify/create-device-container.ts

```diff
@@ -41,6 +42,7 @@ export const createDeviceContainer = (serial: string): Container => {
   deviceContainer.bind(CONTAINER_IDS.deviceScreenStore).to(DeviceScreenStore)
   deviceContainer.bind(CONTAINER_IDS.deviceControlStore).to(DeviceControlStore)
   deviceContainer.bind(CONTAINER_IDS.deviceBySerialStore).to(DeviceBySerialStore)
+  deviceContainer.bind(CONTAINER_IDS.portForwardingService).to(PortForwardingService)
```

Comment:
With the context of the #94 I think a good idea might be to add an interface so that the services can clean up after the device is being done used.